Plugins/VspSourceService/TrackOracleArchiveSource/track_oracle_utils.h

```diff
@@ -34,7 +34,7 @@ namespace track_oracle
   struct track_field_type;
 
   template <typename T>
-  struct track_field_type<track_oracle::track_field<T>&>
+  struct track_field_type<::track_oracle::track_field<T>&>
```

Comment:
@RustyBlue, it built for me without this change, but I'll take your word for it. Anyway, at worst, it shouldn't hurt anything, and it's more consistent with `TRACK_ORACLE_INIT_FIELD`.